RecoEgamma/ElectronIdentification/interface/ElectronMVAEstimatorRun2Fall17.h

```diff
@@ -0,0 +1,103 @@
+#ifndef RecoEgamma_ElectronIdentification_ElectronMVAEstimatorRun2Fall17_H
+#define RecoEgamma_ElectronIdentification_ElectronMVAEstimatorRun2Fall17_H
+
+#include <vector>
+#include <string>
+
+#include "DataFormats/BeamSpot/interface/BeamSpot.h"
+#include "DataFormats/EgammaCandidates/interface/Conversion.h"
+#include "DataFormats/EgammaCandidates/interface/ConversionFwd.h"
+#include "DataFormats/PatCandidates/interface/Electron.h"
+#include "DataFormats/TrackReco/interface/Track.h"
+#include "FWCore/ParameterSet/interface/FileInPath.h"
+#include "RecoEgamma/EgammaTools/interface/AnyMVAEstimatorRun2Base.h"
+#include "RecoEgamma/EgammaTools/interface/ConversionTools.h"
+#include "RecoEgamma/EgammaTools/interface/GBRForestTools.h"
+
+class ElectronMVAEstimatorRun2Fall17 : public AnyMVAEstimatorRun2Base{
+
+ public:
+
+  // Constructor and destructor
+  ElectronMVAEstimatorRun2Fall17(const edm::ParameterSet& conf, bool withIso);
+  ElectronMVAEstimatorRun2Fall17(const std::string &mvaTag, const std::string &mvaName, bool withIso, const std::string &conversionsTag = "reducedEgamma:reducedConversions", const std::string &beamspotTag = "offlineBeamSpot");
+  ~ElectronMVAEstimatorRun2Fall17() override;
+
+  void init(const std::vector <std::string> weightFileNames);
+
+  // Calculation of the MVA value (VID accessor)
+  float mvaValue( const edm::Ptr<reco::Candidate>& particle, const edm::Event&) const override;
+  // Calculation of the MVA value (fwlite-compatible accessor)
+  float mvaValue( const reco::GsfElectron * particle, const edm::Event &) const ;
```

Comment:
@guitargeek this needs to take edm::EventBase in order to be used from FWLite, can you please change it back?

@gpetruc

Reply:
Done!